python/ray/serve/_private/deployment_state.py

```diff
@@ -83,28 +87,45 @@ class DeploymentTargetState:
     info: Optional[DeploymentInfo]
     num_replicas: int
     version: Optional[DeploymentVersion]
+    autoscaling_policy: Optional[AutoscalingPolicy]
     deleting: bool
 
     @classmethod
     def default(cls) -> "DeploymentTargetState":
-        return cls(None, -1, None, False)
+        return cls(None, -1, None, None, False)
 
     @classmethod
     def from_deployment_info(
-        cls, info: DeploymentInfo, *, deleting: bool = False
+        cls,
+        info: DeploymentInfo,
+        autoscaled_num_replicas: int = None,
```

Comment:
can you please add a docstring that describes what this field is and how it interacts with the deployment info?

Reply:
Per Sihan's suggestion, removed it and placed into deployment info